titus-common-server/src/test/java/com/netflix/titus/runtime/clustermembership/endpoint/grpc/ClusterMembershipServerResource.java

```diff
@@ -90,6 +92,20 @@ protected void before() {
                 ReactorClusterMembershipClient.class,
                 ClusterMembershipServiceGrpc.getServiceDescriptor()
         );
+
+        Stopwatch stopwatch = Stopwatch.createStarted();
+        while (true) {
+            try {
+                client.getMembers().block();
```

Comment:
I believe this blocks forever. How about having a maximum block duration such that it retries as well?

Reply:
I will remove this code. The `ClusterMembershipService` junit tests fail when run by Github actions. I will create a separate JIRA ticket to address that. I temporarily disabled all tests for this class.